wrench/Cargo.toml

```diff
@@ -18,15 +18,15 @@ image = "0.12"
 clap = { version = "2", features = ["yaml"] }
 lazy_static = "0.2"
 yaml-rust = { git = "https://github.com/vvuk/yaml-rust", features = ["preserve_order"] }
-serde_json = "0.9"
+serde_json = "1.0"
 time = "0.1"
 crossbeam = "0.2"
 osmesa-sys = { version = "0.1.2", optional = true }
 osmesa-src = { git = "https://github.com/servo/osmesa-src", optional = true }
 webrender = {path = "../webrender"}
 webrender_traits = {path = "../webrender_traits"}
-serde_derive = "0.9"
-serde = "0.9"
+serde_derive = "1.0"
+serde = "1.0"
```

Comment:
Can you use the `derive` feature of `serde` instead of explicitly depending on `serde_derive`?